README.md

```diff
@@ -1,56 +1,60 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Countries
 
-> Description the project.
+> Countries is a small we app which gets south asian countries details from an api and shows its' details like name, flag, lag, long and population.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript
+- React
+- Redux
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://astonishing-piroshki-2b21e0.netlify.app/)
 
 
 ## Getting Started
```

Comment:
- [ ] There should be a video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.